### PR TITLE
Add post documentation lockdown survey link

### PR DESCRIPTION
**Pull request by JoelMarcey**
We did a survey prior to the doc lockdown to get an idea of what we should focus on. Now we have a yes/no style survey to ensure that we met the user needs when it came to our start of improving the docs.

This diff adds the survey link to the bottom of each page. And we will cherry-pick it into 0.29 as well, which will be the first official release with the new doc content

> **NOTE**: The content via the links in this survey will not be valid until 0.29 is actually released. So I have not enabled the questions for the survey until then.

**Test Plan:**

http://localhost:8079/react-native/docs/getting-started.html

screenshot 2016-07-01 17 35 17

### CODE REVIEW

**Comment by facebook-github-bot-8** [comment]
By analyzing the blame information on this pull request, we identified @Andreyco and @caabernathy to be potential reviewers.

File: website/layout/DocsLayout.js [hunk 1]
```diff
@@ -48,6 +48,15 @@ var DocsLayout = React.createClass({
               {metadata.previous && <a className="docs-prev" href={'docs/' + metadata.previous + '.html#content'}>&larr; Prev</a>}
               {metadata.next && <a className="docs-next" href={'docs/' + metadata.next + '.html#content'}>Next &rarr;</a>}
             </div>
+            <div className="survey">
+              <div className="survey-image" />
+              <p>
+                We just had a lockdown to improve the React Native documentation. Your responses to this yes/no style survey will help us gauge whether we moved in the right direction with the improvements. Thank you!
```
Comment (hramos) [inline website/layout/DocsLayout.js]: Is the term 'lockdown' used in a similar manner outside of FB? Maybe go with similar wording as the [blog post](https://github.com/lacker/react-native/blob/e5149c8c1fda8dbbb2ff0209303eca57447bf235/blog/2016-07-05-toward-better-documentation.md):

> "Recently we've been working hard to make the docs better based on your feedback. Your responses to this yes/no style survey will help us gauge whether we moved in the right direction with the improvements. Thank you!"

**Comment by hramos** [comment]
LGTM. Minor nit regarding the use of lockdown but I don't feel too strongly about it.

**Comment by JoelMarcey** [comment]
@hramos Ah - good point. Fixed.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Something went wrong when importing this pull request. Please cc someone from the team at fb to help with importing this.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/578897528957141/int_phab) to review.

**Pull request closed without merging**